### PR TITLE
Made min password length same across the application

### PR DESCRIPTION
**Pull request by pravesh-sharma**

### CODE REVIEW

**Comment by adityatoshniwal** [comment]
@pravesh-sharma The PR is in draft state.

File: web/config.py [hunk 1]
```diff
@@ -192,6 +192,9 @@
 # Hashing algorithm used for password storage
 SECURITY_PASSWORD_HASH = 'pbkdf2_sha512'
 
+# Minimum password length
+SECURITY_PASSWORD_LENGTH_MIN = 6
```
Comment (adityatoshniwal) [inline web/config.py:196]: Will this apply when adding new user from users dialog?

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

File: web/config.py (hunk 1 above)
Comment (pravesh-sharma) [reply web/config.py:196]: users dialog is already having validation on 6 characters.

Comment (adityatoshniwal) [reply web/config.py:196]: OK.

**Pull request merged**